tests/test_dice_interface/test_dice_genetic.py

```diff
@@ -177,7 +177,43 @@ def test_predict_custom(self, desired_class, sample_custom_query_2, mocker):
         mocker.patch('dice_ml.model_interfaces.base_model.BaseModel.get_output', return_value=[[0, 0.5, 0.5]])
         custom_preds = self.exp._predict_fn_custom(sample_custom_query_2, desired_class)
         assert custom_preds[0] == desired_class
-
+    
+    # Testing if the shapes of the predictions are correct for multiclass classification
+    @pytest.mark.parametrize(("desired_class", "method"), [(1, "genetic")])
+    def test_multiclass_nn(self, desired_class, method):
+        pytest.importorskip("torch")
+        pytest.importorskip("sklearn")
```

Comment:
Why are the above two lines needed? I think we install these dependencies before executing the tests.

Reply:
I put the lines to make sure sklearn and pytorch were imported, but you're right. I'll remove them in the next commit.